backend/build.gradle.kts

```diff
@@ -58,6 +59,8 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
 
 tasks.withType<Test> {
     useJUnitPlatform()
+    minHeapSize = "512m"
+    maxHeapSize = "1024m"
```

Comment:
Vážne potrebujeme tak veľa pamäte na test? :)

Reply:
Minimalni muzeme ponizit, ale ty JPA testy s bootem aplikace jsou nejaky nenazrany a 512MB jim nestaci.

Reply:
Ak to beží iba lokálne, tak ok. Inak (cloud, platba za RAM) by sme sa na RAM museli pozrieť. Stále to je divné :).